tldw_Server_API/tests/Sharing/test_sharing_endpoints.py

```
@@ -0,0 +1,300 @@
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+
+from tldw_Server_API.app.api.v1.endpoints.sharing import router
+from tldw_Server_API.app.core.AuthNZ.User_DB_Handling import User
+
+pytestmark = pytest.mark.unit
+
+
+class _TestUser(User):
+    """Subclass with team/org membership for testing shared-with-me."""
+    team_ids: list[int] = []
```

Ruff (0.15.6)

[warning] Mutable default value for class attribute

(RUF012)